Updating ubuntu section (#1662)
pyansys/pymapdl: doc/source/troubleshoot/troubleshoot.rst

    yum install openssl openssh-clients mesa-libGL mesa-libGLU motif libgfortran

.. _installing_mapdl_on_ubuntu:

Ubuntu
------

> On Ubuntu 22.04, it is needed to install the following:

Reviewer: ```suggestion
On Ubuntu 22.04, use this code to install the needed dependencies:
```

.. code::

    apt-get update

    # Install dependencies
    apt-get install -y \
    openssh-client \
    libgl1 \
    libglu1 \
    libxm4 \
    libxi6

> This takes care of everything except for ``libxp6`` for which you need the following:

Reviewer: ```suggestion
The preceding code takes care of everything except for ``libxp6``, which you must install
using this code:
```

.. code:: bash

    # This is a workaround
    # Source: https://bugs.launchpad.net/ubuntu/+source/libxp/+bug/1517884
    apt install -y software-properties-common
    add-apt-repository -y ppa:zeehio/libxp
    apt-get update
    apt-get install -y libxp6

Ubuntu 20.04 and older
----------------------

> If you are using Ubuntu 16.04, you can install that simply with ``sudo apt
> install libxp6``. However, on Ubuntu 18.04+ up to 20.04, you must manually

Reviewer: ```suggestion
If you are using Ubuntu 16.04, you can install ``libxp16`` with this code:

.. code:: bash

   sudo apt install libxp6. 
   
However, on Ubuntu 18.04 up to 20.04, you must manually
```
